pylint/pyreverse/printer.py

```diff
@@ -84,6 +86,28 @@ def emit_edge(
     ) -> None:
         """Create an edge from one node to another to display relationships."""
 
+    @staticmethod
+    def _get_method_arguments(method: astroid.FunctionDef) -> List[str]:
+        if method.args.args:
+            arguments: List[astroid.AssignName] = [
+                arg for arg in method.args.args if arg.name != "self"
+            ]
+        else:
+            arguments = []
+
+        annotations = dict(zip(arguments, method.args.annotations[1:]))
+        for arg in arguments:
+            annotation_label = ""
+            ann = annotations.get(arg)
+            if ann:
+                annotation_label = get_annotation_label(ann)
+            annotations[arg] = annotation_label
+
+        return [
+            f"{arg.name}: {ann}" if ann else f"{arg.name}"
+            for arg, ann in annotations.items()
+        ]
+
```

Comment:
This method is used by both ``DotPrinter`` and ``PlantUmlPrinter``. We could probably add this feature for VCG as well if somebody needs it.